R/module_nested_tabs.R

```diff
@@ -306,29 +301,22 @@ srv_nested_tabs.teal_module <- function(id, datasets, modules, is_module_specifi
   # list of reactive filtered data
   data <- sapply(
     datanames,
-    function(x) eventReactive(trigger_data(), datasets$get_data(x, filtered = TRUE)),
+    function(x) isolate(datasets$get_data(x, filtered = TRUE)),
```

Comment:
Function is run in a reactive context (line 246) so you can remove isolate (you'll have just two tests to fix)

```suggestion
    function(x) isolate(datasets$get_data(x, filtered = TRUE)),
```